assets/js/util/index.js

```diff
@@ -38,6 +38,7 @@ export * from './urls';
 export * from './is-valid-numeric-id';
 export * from './isnumeric';
 
+global._gsktag = trackEvent;
```

Comment:
Let's add some whitespace here to separate this more from the comment below…

```suggestion
global._gsktag = trackEvent;

```

That said: why place this global here? This seems a bit… "buried"/hidden in the codebase for this type of change to be made, especially with no comment explaining it or what it does.

Can you add a comment explaining what this is and why it's here?

It might also be good to put in a more dedicated place related to `trackEvent`, etc. 🤔

Reply:
> That said: why place this global here?

@tofumatt thanks for picking this up, that was left over from debugging the trackEvent via a global that could be accessed in console I believe, I have removed it and I am still observing the internal tracking functioning correctly.

Back over to you.

Reply:
From the IB :)

> [Remove WIP test function](https://github.com/google/site-kit-wp/compare/poc/track-event-ga4#diff-35e7b36727377afdfe72372f2fdc85d4d74f6a423a820ad573c77b3c07a1341b)